### PR TITLE
Human samples for star test

### PR DESCRIPTION
**Pull request by HudoGriz**
## Checklist

* [x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [ ] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [x] All inputs are used in process.
* [ ] All output fields have their ``re-save`` calls.

In the testing process for the STAR aligner, were mouse sample aligned to human index, which resulted in zero mappings. By changing the samples to human (GRCh38.p12_NCBIchr21_R1.fq.gz and its counterpart) we get a positive amount of mapped reads. The additional files are required for comparison of output.

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -70,6 +70,7 @@ Changed
 - Use ``resolwebio/common:1.3.1`` as a base Docker image for the
   ``resolwebio/wgbs:1.2.0`` Docker image
 - Support MultiQC reports in ChIP-seq workflows
+- Use human samples for ``alignment-star`` testing in ``test_alignment.py``
```
Comment (romunov) [inline docs/CHANGELOG.rst]: Break the line to be no more than 72 characters.

Comment (HudoGriz) [reply docs/CHANGELOG.rst]: I took samples from `workflow_bbduk_star_htseq` and this is the new outcome:
```
N_unmapped	0	0	0
N_multimapping	0	0	0
N_noFeature	70	77	80
N_ambiguous	0	0	0
ENSG00000279493	17	10	7
```

File: resolwe_bio/tests/processes/test_alignment.py [hunk 2]
```diff
@@ -228,29 +281,77 @@ def test_star(self):
...
+            exp,
+            'exp_set',
+            os.path.join('test_star', 'output', 'star_out_exp_set.txt.gz'),
+            compression='gzip'
+        )
+        self.assertJSON(
+            exp,
+            exp.output['exp_set_json'],
+            '',
+            os.path.join('test_star', 'output', 'star_exp_set.json.gz')
```
Comment (jkokosar) [inline resolwe_bio/tests/processes/test_alignment.py]: The file with the same name is tested in two places. For such cases, you cold simplify the lines by defining a variable first, and then using this where appropriate, e.g.
```
exp_set_json = os.path.join('test_star', 'output', 'star_exp_set.json.gz')

self.assertJSON(exp, exp.output['exp_set_json'], '', exp_set_json)
```

File: resolwe_bio/tests/processes/test_alignment.py [hunk 3]
```diff
@@ -116,27 +118,41 @@ def test_star_index(self):
...
-            reads = self.prepare_reads(['SRR2124780_1 1k.fastq.gz'])
-            paired_reads = self.prepare_paired_reads(mate1=['SRR2124780_1 1k.fastq.gz'],
-                                                     mate2=['SRR2124780_2 1k.fastq.gz'])
-            annotation = self.prepare_annotation(fn='HS chr21_short.gtf.gz', source='ENSEMBL',
-                                                 species='Homo sapiens', build='GRCh38_ens90')
+            reads = self.prepare_reads(
+                [os.path.join('test_star', 'input', 'hs_single bbduk_star_htseq_reads_single.fastq.gz')])
+            paired_reads = self.prepare_paired_reads(
+                mate1=[os.path.join('test_star', 'input', 'hs_paired_R1 workflow_bbduk_star_htseq.fastq.gz')],
+                mate2=[os.path.join('test_star', 'input', 'hs_paired_R2 workflow_bbduk_star_htseq.fastq.gz')]
```
Comment (jvrakor) [inline resolwe_bio/tests/processes/test_alignment.py]: Missing trailing comma.

File: resolwe_bio/tests/processes/test_alignment.py [hunk 4]
```diff
@@ -116,27 +118,41 @@ def test_star_index(self):
...
+            )
+            annotation = self.prepare_annotation(
+                os.path.join('test_star', 'input', 'hs annotation.gtf.gz'),
+                source='ENSEMBL',
+                species='Homo sapiens',
+                build='GRCh38_ens90'
```
Comment (jvrakor) [inline resolwe_bio/tests/processes/test_alignment.py]: Missing trailing comma.

File: resolwe_bio/tests/processes/test_alignment.py [hunk 5]
```diff
@@ -116,27 +118,41 @@ def test_star_index(self):
...
             star_index_fasta = self.run_process('upload-fasta-nucl', inputs)
-            inputs = {'annotation': annotation.id, 'genome2': star_index_fasta.id}
 
+            inputs = {'annotation': annotation.id, 'genome2': star_index_fasta.id}
             star_index = self.run_process('alignment-star-index', inputs)
 
-            star_index_wo_annot = self.run_process('alignment-star-index', {'genome2': star_index_fasta.id})
+            star_index_wo_annot = self.run_process(
+                'alignment-star-index',
+                {'genome2': star_index_fasta.id}
```
Comment (jvrakor) [inline resolwe_bio/tests/processes/test_alignment.py]: Missing trailing comma.

File: docs/CHANGELOG.rst [hunk 6]
```diff
@@ -70,6 +70,8 @@ Changed
 - Use ``resolwebio/common:1.3.1`` as a base Docker image for the
   ``resolwebio/wgbs:1.2.0`` Docker image
 - Support MultiQC reports in ChIP-seq workflows
+- Use human samples for ``alignment-star`` testing in
+  ``test_alignment.py``
```
Comment (jvrakor) [inline docs/CHANGELOG.rst]: Probably no need to tell which file is being changed.

**Pull request merged**